cetmix_tower_server/models/cx_tower_command.py

```diff
@@ -128,6 +128,12 @@ class CxTowerCommand(models.Model):
         help="Set the following status if command finishes with success. "
         "Leave 'Undefined' if you don't need to update the status",
     )
+    no_split_for_sudo = fields.Boolean(
+        string="No Split for sudo",
+        default=False,
```

Comment:
`False` is the default value for the Boolean fields, no need to put this value explicitly.